docs/man/Dockerfile.5.md

```diff
@@ -143,6 +143,21 @@ A Dockerfile is similar to a Makefile.
   **CMD** executes nothing at build time, but specifies the intended command for
   the image.
 
+**LABEL**
+  -- `LABEL <key>[=<value>] [<key>[=<value>] ...]`
+  The **LABEL** instruction adds metadata to an image. A **LABEL** is a
+  key-value pair. To include spaces within a **LABEL** value, use quotes and
+  blackslashes as you would in command-line parsing.
+
+  ```
+  LABEL "com.example.vendor"="ACME Incorporated"
+  ```
+
+  An image can have more than one label. To specify multiple labels, separate each
+  key-value pair by a space.
```

Comment:
what happens if my Dockerfile has more than one `LABEL` command?

how do `LABEL`s in `FROM` images affect the end result - are they additive?